hostplumber/pkg/ovs-docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.18
```

Comment:
This is the key change here that fixes the issues building on Teamcity. Same as https://github.com/platform9/luigi/pull/138

Reply:
> Stick to Alpine 3.16 in Hostplumber : Backport to v0.5.4 branch
> 
> Reason : Hostplumber build uses `libexecinfo` and `libexecinfo-dev` which has been removed in Alpine 3.17 - build fails in Teamcity with higher version of Alpine [DataDog/dd-trace-php#1824](https://github.com/DataDog/dd-trace-php/issues/1824)
> 
> TODO: Can look into workarounds later to be able to support alpine 3.17 or higher YMMV : [awslabs/aws-lambda-cpp#124 (comment)](https://github.com/awslabs/aws-lambda-cpp/issues/124#issuecomment-1829207396)
> 
> ## Teamcity builds :
> * luigi-plugins : https://teamcity.platform9.horse/buildConfiguration/Pf9project_Platform9ComponentsForKubernetes_LuigiPlugins/3167489
> * hostplumber : https://teamcity.platform9.horse/buildConfiguration/Pf9project_Platform9ComponentsForKubernetes_Hostplumber/3167492